CONTRIBUTING.md

```diff
@@ -56,9 +56,8 @@ Try these useful commands below:
 
 * Format code automatically: `bash tools/run_docker.sh -c 'make code-format'`
```

Comment:
The makefile is now removed so this needs to be updated.

Reply:
It's done in #1028 and I don't want to have conflicts :(